libgstc/python/meson.build

```diff
@@ -1 +1,25 @@
-run_command('pip3', 'install', '.')
+pymod = import('python')
+python = pymod.find_installation(
+  get_option('with-python-version'),
+  required : get_option('enable-python').enabled(),
+  disabler : true
+)
+
+pythonver = python.language_version()
+if pythonver.version_compare('<3.7')
+  error('Python @0@ is not supported anymore, please port your code to python3.7 or newer.'.format(python.language_version()))
+endif
```

Comment:
We need to relax this a bit. Some Jetson boards still ship will lower Python versions. For example:
*Jetson Xavier AGX - Jetpack 4.6*
```
nvidia@localhost:~$ python3 --version
Python 3.6.9
```
Is Python officially dropping maintenance for pre 3.7?

Reply:
Oh, that may have been due to a feature(`asynccontextmanager`) requiring 3.7 that I used in #252, I can probably rework that for python 3.6 compatibility if needed.

Reply:
Rebased and reverted minimum python version to `3.5`, I'll re-base the asyncio pull request on top of this one after. Is python3.6 the minimum version we need to support?

Reply:
I would drop it to plain 3, and add a special requirement for the asyncio client independently (in that branch). We can't drop the original client for backwards compatibility reasons, so I'm thinking a more model alternative of the client.

Reply:
> I would drop it to plain 3

Well it's currently minimum `3.5` here, is that requirement accurate?:
https://github.com/RidgeRun/gstd-1.x/blob/fa523d54628b04fe3242ba126ab069e0b35a618b/libgstc/python/setup.py#L50

> We can't drop the original client for backwards compatibility reasons, so I'm thinking a more model alternative of the client.

Oh, I guess there's clients that expect a stable API across library versions? I can probably just add some compatibility shims for API comaptibility.